### PR TITLE
Removed  eslint-mocha and eslint-config-node-services.

### PR DESCRIPTION
**Pull request by hknustwmf**

### CODE REVIEW

File: maintenance/lib/index.js [hunk 1]
```diff
@@ -29,5 +27,5 @@ function getConfig(config) {
 }
 
 module.exports = {
-    getConfig: getConfig,
+    getConfig: getConfig
```
Comment (d00rman) [inline maintenance/lib/index.js]: This can be simplified as just `getConfig`

File: scripts/RashomonDumpImporter.js [hunk 2]
```diff
@@ -21,7 +21,7 @@ function testWrites () {
 		maxConcurrency = 50;
 	http.globalAgent = false;
 
-	reader.on( 'revision', function ( revision ) {
+	reader.on('revision', function (revision) {
```
Comment (d00rman) [inline scripts/RashomonDumpImporter.js]: `() => {}`

File: scripts/RashomonDumpImporter.js [hunk 3]
```diff
@@ -86,7 +86,7 @@ function testWrites () {
 		// send it off
 		doPost();
 	});
-	reader.on('end', function() {
+	reader.on('end', function () {
```
Comment (d00rman) [inline scripts/RashomonDumpImporter.js]: `() => {}`

File: scripts/direct-request.js [hunk 4]
```diff
@@ -60,8 +55,8 @@ var opts = {
 var exp_method = false;
 var exp_data = false;
 var exp_fname = false;
-args.forEach(function(arg, index, array) {
-    switch(arg) {
+args.forEach(function (arg, index, array) {
```
Comment (d00rman) [inline scripts/direct-request.js]: `() => {}`

File: scripts/direct-request.js [hunk 5]
```diff
@@ -101,34 +96,32 @@ args.forEach(function(arg, index, array) {
...
 
-
 makeClient({
     log: console.log,
     conf: {
         hosts: ['localhost']
     }
 })
-.then(function(db) {
+.then(function (db) {
```
Comment (d00rman) [inline scripts/direct-request.js]: Here and below:
`() => {}`

File: scripts/dumpReader.js [hunk 6]
```diff
@@ -12,7 +12,7 @@ util.inherits(DumpReader, events.EventEmitter);
 /**
  * @param {Stream} stream input stream to read XML from
  */
-DumpReader.prototype.makeParser = function() {
+DumpReader.prototype.makeParser = function () {
```
Comment (d00rman) [inline scripts/dumpReader.js]: Here and below: `() => {}`

File: maintenance/lib/index.js (hunk 1 above)
Comment (hknustwmf) [reply maintenance/lib/index.js]: Done

File: scripts/RashomonDumpImporter.js (hunk 3 above)
Comment (hknustwmf) [reply scripts/RashomonDumpImporter.js]: Done

File: scripts/direct-request.js (hunk 4 above)
Comment (hknustwmf) [reply scripts/direct-request.js]: Done

File: scripts/dumpReader.js (hunk 6 above)
Comment (hknustwmf) [reply scripts/dumpReader.js]: Done

File: scripts/direct-request.js (hunk 5 above)
Comment (hknustwmf) [reply scripts/direct-request.js]: Done

File: maintenance/list_tables.js [hunk 7]
```diff
@@ -92,6 +90,7 @@ return client.execute('SELECT keyspace_name,table_name FROM system_schema.tables
         for (const i of tuples) {
             const keyspace = i[0];
             const table = i[1];
+            // eslint-disable-next-line no-console
```
Comment (Pchelolo) [inline maintenance/list_tables.js]: We can actually create a separate .eslintrc in the root of the maintenance folder and not use per-line overrides. I think it makes it more visible

File: scripts/RashomonDumpImporter.js [hunk 8]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env node
-/*jslint node: true */
-"use strict";
+/* jslint node: true */
```
Comment (Pchelolo) [inline scripts/RashomonDumpImporter.js]: This line is the remaining of jslint times, can be removed.

**Comment by Pchelolo** [comment]
A couple of super-minor nits, otherwise LGTM

**Comment by Pchelolo** [comment]
There's also a .jshintignore file in this repo, let's delete it as a part of this pull request.

File: maintenance/list_tables.js (hunk 7 above)
Comment (hknustwmf) [reply maintenance/list_tables.js]: Agreed

File: scripts/RashomonDumpImporter.js (hunk 2 above)
Comment (hknustwmf) [reply scripts/RashomonDumpImporter.js]: Removed

**Pull request merged**